go.mod

```diff
@@ -1,11 +1,10 @@
 module github.com/maxmind/geoip2-csv-converter
 
-go 1.17
+go 1.18
```

Comment:
Regarding the linter issue : https://github.com/golangci/golangci-lint-action/issues/434#issuecomment-1088100765

I think it is OK to ignore the lint check for now.